src/patternfly/components/Card/themes/dark/card.scss

```diff
@@ -10,4 +11,8 @@
     --pf-c-card--m-selectable-raised--focus--BoxShadow: var(--pf-global--BoxShadow--lg);
     --pf-c-card--m-selectable-raised--active--BoxShadow: var(--pf-global--BoxShadow--lg);
   }
+
+  .pf-c-page__main-section[class*="pf-m-dark-"] .pf-c-card {
```

Comment:
This has the potential of styling cards we don't intend on styling, like if they're used in some other component and not directly against the background color of the main section. That said, I hadn't made any sort of considerations for the `.pf-m-dark-` page section backgrounds with the dark theme work, so I wonder how often they're used and if this is likely to even be a problem.